src/Microsoft.ML.Data/DataView/SimpleRow.cs

```diff
@@ -204,8 +205,9 @@ public static Schema Create(IExceptionContext ectx, params KeyValuePair<string,
             Contracts.CheckValueOrNull(ectx);
             ectx.CheckValue(columns, nameof(columns));
 
-            var cols = columns.Select(kvp => new Schema.Column(kvp.Key, kvp.Value, null));
-            return new Schema(cols);
+            var builder = new SchemaBuilder();
+            builder.AddColumns(columns.Select(kvp => new ML.Data.Schema.DetachedColumn(kvp.Key, kvp.Value, null)));
```

Comment:
>DetachedColumn(kvp.Key, kvp.Value, null) [](start = 72, length = 40)

I don't like idea of passing null to constructor, can we have constructor with two parameters which imply what metadata is absent?